### PR TITLE
[AR] Translation of 06-2.md

### PR DESCRIPTION
**Pull request by AliEl-01**
i this PR i have fixed some previous issue in 06-2.md and also translating the title in 06-1.md

### CODE REVIEW

File: docs/ar/week06/06-1.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
+---
+lang-ref: ch.06-1
+lang: ar
+lecturer: Yann LeCun
+title: تطبيقات الشبكات الالتفافية
+authors: Shiqing Li, Chenqin Yang, Yakun Wang, Jimin Tan
+date: 2 مارس 2020
```
Comment (Atcold) [inline docs/ar/week06/06-1.md:7]: Restore previous date.

File: docs/ar/week06/06-1.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+translation date: 1 Sep 2020
```
Comment (Atcold) [inline docs/ar/week06/06-1.md:8]: The field is called `translation-date`.

File: docs/ar/week06/06-2.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
+---
+lang-ref: ch.06-2
+lang: ar
+lecturer: Yann LeCun
+title: الشبكات العصبية التكرارية، وحدات البوابات المتكررة، الذاكرة القصيرة المدى المطولة، الانتباه، تسلسل/تسلسل و شبكة الذاكرة.
+authors: Jiayao Liu, Jialing Xu, Zhengyang Bian, Christina Dominguez
+date: 2 مارس 2020
```
Comment (Atcold) [inline docs/ar/week06/06-2.md:7]: `translator:` and `translation-date:` are missing.
`data:` field got corrupted.

**Review by Atcold — CHANGES_REQUESTED** [review]
A few changes, plz.

File: docs/ar/week06/06-1.md (hunk 1 above)
Comment (AliEl-01) [reply docs/ar/week06/06-1.md:7]: I didn't get it!!?

**Comment by AliEl-01** [comment]
Ok I will

File: docs/ar/week06/06-2.md (hunk 3 above)
Comment (AliEl-01) [reply docs/ar/week06/06-2.md:7]: which data is corrupted???

**Comment by AliEl-01** [comment]
hope this changes will fix the previous problem!

**Review by Atcold — APPROVED** [review]
Sweet, perfect!

**Pull request merged**